backend/api/communication/src/main/java/co/airy/core/api/communication/dto/Conversation.java

```diff
@@ -21,7 +20,7 @@
 @AllArgsConstructor
 public class Conversation implements Serializable {
     private Long createdAt;
-    private Message lastMessage;
+    private MessageWrapper lastMessage;
```

Comment:
since we're at it I would rename to `lastMessageWrapper` (the caller code would benefit in clarity)